lib/inspec/dependencies/requirement.rb

```diff
@@ -1,6 +1,5 @@
 # encoding: utf-8
 require 'inspec/cached_fetcher'
-require 'inspec/dependencies/dependency_set'
```

Comment:
Huh...not sure how removing this doesn't break anything.

Reply:
Trace the requires.  

1. inspec/profile.rb requires inspec/dependencies/dependency_set
2. inspec/dependencies/dependency_set requires inspec/dependencies/requirement
3. inspec/dependencies/requirement requires inspec/dependencies/dependency_set, triggering a circular dependency warning.  I removed this require to break the loop.

Reply:
Ah! Great catch.